src/app/components/media/SuperAdminControls.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import { FormattedMessage } from 'react-intl';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  remove: {
+    color: '#F44336',
+    borderColor: '#F44336',
+    border: '2px solid',
+  },
+  apply: {
+    color: '#4CAF50',
+    borderColor: '#4CAF50',
+    border: '2px solid',
+  },
+});
```

Comment:
Since this is a new component, let's implement the styles using CSS like we've been doing with more recent components.

Reply:
When doing that, don't hard-code color: Use variables, please.

Reply:
Reference, for example: https://github.com/meedan/check-web/blob/develop/src/app/components/feed/FeedBlankState.js#L7.

Reply:
@melsawy If you like we could pair on this and I can show you how the new CSS stuff works, or I could make a PR on this PR and then you would have a reference forever (which I would also put in Confluence). Let me know if one or the other sounds good, or if you've already done this and just haven't gotten around to this PR of course that's fine too!

Reply:
@dariusk I moved the CSS stuff to a new file `SuperAdminControls.module.css` but still did not work so I'll ping you for a pairing to fix this one